internal/explorer/server.go

```diff
@@ -175,17 +176,29 @@ func (a *App) getNode(w http.ResponseWriter, r *http.Request) {
 func (a *App) getNodeStatus(w http.ResponseWriter, r *http.Request) {
 	enableCors(&w)
 
+	response := NodeStatus{Status: "loading"}
+
 	nodeID := mux.Vars(r)["node_id"]
-	_, err := a.getNodeData(nodeID, false)
+	value, err := a.GetRedisKey(fmt.Sprintf("GRID3NODE:%s", nodeID))
 	if err != nil {
-		w.Header().Add("Content-Type", "application/json")
-		w.WriteHeader(http.StatusOK)
-		w.Write([]byte("{\"status\": \"down\"}"))
+		response.Status = "loading"
```

Comment:
this doesn't separate errnotfound and other kinds of errors. when a node is not found it gets deleted from redis, so a non existent node would show state as loading?

Reply:
No, that's until nodes loads, when the first loop complete it will be either up or down